dgbpy/torch_classes.py

```diff
@@ -714,18 +715,17 @@ def __repr__(self):
         return f'{d}'
 
 class RandomFlip():
-    def __init__(self, p=0.55):
+    def __init__(self, p=0.3):
         self.p = p
 
-    def update_pars(self, transform_dict):
-        self.p = float(transform_dict['p'])
+    def transformLabel(self, info):
+        return dgbhdf5.isImg2Img(info)
 
-    def __call__(self, image, label, image_only = False):
+    def __call__(self, image=None, label=None):
         if self.p > np.random.uniform(0,1):
-            if image_only:
+            if isinstance(label, type(None)):
```

Comment:
Would "if not label:" have the same result?

Reply:
I think `if not label` might not work with numpy arrays..
Also, a label image with only values of '0' might result into issues.